### ISSUE TITLE
 Cannot find electron v0.22.0 from GitHub null

### ISSUE THREAD

**Issue by gentunian** [atom/atom#8525]
Hey there,

I'm trying to build atom from sources for fedora. I've tried the `rpm` prebuilt binary but it is not working. So I decided to build from sources following the instructions in this repo, this is what I got:

```
$ script/build
Node: v0.10.39
npm: v2.13.3
Installing build modules...
Installing apm...
Installing modules ✓
Deduping modules ✓
Running "download-atom-shell" task

Running "download-atom-shell-chromedriver" task
>> Cannot find electron v0.22.0 from GitHub null
Aborted due to warnings.
```

Any ideas what I could be doing wrong?

By the way, `npm` version differs from the one reported above:

```
$ npm -v
1.4.28
```

**Comment by 50Wliu** [comment]
This is happening on Travis as well.

**Comment by gentunian** [comment]
I'm sorry, I didn't notice the Travis tag.

**Comment by gentunian** [comment]
If useful, this is the output with `--stack` added:

```
script/build 
Node: v0.10.39
npm: v2.13.3
Installing build modules...
Installing apm...
Installing modules ✓
Deduping modules ✓
Running "download-atom-shell" task

Running "download-atom-shell-chromedriver" task
>> Cannot find electron v0.22.0 from GitHub null
Error: Task "download-atom-shell-chromedriver" failed.
  at Task.<anonymous> (atom/build/node_modules/grunt/lib/util/task.js:205:15)
  at [object Object]._onTimeout (atom/build/node_modules/grunt/lib/util/task.js:241:33)
  at Timer.listOnTimeout [as ontimeout] (timers.js:121:15)

Aborted due to warnings.
```

**Comment by htmlguy** [comment]
Having the same issue on trisquel (ubuntu-based distro) (32-bit):
I followed the ubuntu instructions at https://github.com/atom/atom/blob/master/docs/build-instructions/linux.md, and get the same output for script/build.
This is a new git clone, happened just now. It has worked before on an identical system.

**Comment by mnquintana** [comment]
This seems like some kind of temporary network error - hopefully it'll resolve itself sometime today.

**Comment by maxbrunsfeld** [comment]
This is actually a bug in our build infrastructure, specifically https://github.com/atom/node-github-releases.

**Comment by maxbrunsfeld** [comment]
@gentunian could you pull from master and try again?

**Comment by htmlguy** [comment]
It works from master branch now.

**Comment by gentunian** [comment]
Excellent @maxbrunsfeld it's building now.

**Comment by rkubik** [comment]
Was having the same issue. Solved by pulling in commit 0310ccb.
